litmus-portal/frontend/src/components/QuickActionCard/index.tsx

```diff
@@ -71,6 +72,14 @@ const QuickActionCard = () => {
               {t('quickActionCard.readDocs')}
             </a>
           </QuickActionItems>
+          <QuickActionItems>
+            <div className={classes.imgDiv}>
+              <img src="/icons/docs.png" alt="docs" />
```

Comment:
```suggestion
              <img src="./icons/docs.png" alt="docs" />
```
Add a dot before image path in src.